data/schema.yaml

```diff
@@ -344,12 +371,20 @@ components:
             id:
               type: string
               example: 'p01jvmg5'
+            imageCopyright:
+              type: string
+              example: 'BBC'
             imageUrl:
               type: string
               example: 'ichef.test.bbci.co.uk/images/ic/$recipe/p01jvmms.jpg'
             subType:
               type: string # probably should be an enum in the future
               example: 'clip'
+            syndication:
+              type: object
+              properties:
+                destinations:
+                  type: array
```

Comment:
Could this have an example?

Reply:
I haven't found an example that isn't empty, maybe @Sareh knows where we might find one?